backend/dataall/modules/worksheets/services/worksheet_service.py

```diff
@@ -1,5 +1,10 @@
 import logging
 
+from dataall.core.resource_threshold.db.resource_threshold_repositories import ResourceThresholdRepository
+from dataall.modules.worksheets.aws.glue_client import GlueClient
```

Comment:
GlueClient: unused import

Reply:
only unused because I forgot to uncomment `run_nlq` method - should be used now 😄